modules/installation-guide/partials/proc_importing-external-or-additional-tls-certificates.adoc

```diff
@@ -0,0 +1,218 @@
+
+
+[id="importing-external-or-additional-tls-certificates_{context}"]
+= Importing external or additional TLS certificates to {prod-short}
+
+Network communications between the components of a {prod-short} installation and the started workspaces, are all secured through the TLS protocol, and thus require the use of trusted certificate authorities. 
+
+In some cases it can be necessary to add TLS certificates to the {prod-short} installation, so that every {prod-short} component will consider them as signed by a trusted CA.
+
+Typical cases that may require this addition are:
+
+* when the underlying Kubernetes cluster uses TLS certificates signed by a CA that is not trusted,
+* when {prod-short} server or workspace components connect to external services such as {identity-provider} or a Git server that use TLS certificates signed by an untrusted CA.
+
+To store those certificates, {prod-short} uses a dedicated ConfigMap. Its default name is `ca-certs` but {prod-short} allows configuring its name.
+On OpenShift, the {prod-short} operator even injects cluster trusted certificates into this ConfigMap automatically.
+
+== At {prod-short} installation time
+
+.Prerequisites
+
+* The `{orch-cli}` tool is available.
+* You are ready to create `CheCluster` custom resource.
+
+.Procedure
+
+. Save the certificates you need to import, to a local file system.
++
+[CAUTION]
+====
+* Certificate files are typically stored as Base64 ASCII files, such as `.pem`, `.crt`, `.ca-bundle`. But, they can also be binary-encoded, for example, as `.cer` files. All Secrets that hold certificate files should use the Base64 ASCII certificate rather than the binary-encoded certificate.
+* {prod-short} already uses some reserved file names to automatically inject certificates into the ConfigMap, so you should avoid using the following reserved file names to save your certificates:
+  ** `ca-bundle.crt`
+  ** `ca.crt`
+====
```

Comment:
The paragraph above is repeated multiple times in this article. Can we move it to a different doc component and reference it multiple times?

Reply:
Fixed in commit https://github.com/eclipse/che-docs/pull/1598/commits/3f481ae07d585dc12a78fe6300ba5daea7659c93